cookiecutter.json

```diff
@@ -10,10 +10,22 @@
   "_copy_without_render": [
     "clients/vue3/src/views/dashboard",
     "clients/vue3/src/components",
-    "clients/react/src/pages",
+    "clients/react/src/components",
+    "clients/react/src/pages/app-or-auth.tsx",
```

Comment:
Does someone know a way to do "all files in `react/src/pages` except `log-in.tsx` and `sign-up.tsx` ?

I tried doing
```
//...
"clients/react/src/pages",
"!clients/react/src/pages/log-in.tsx",
"!clients/react/src/pages/sign-up.tsx"
//...
```

But that did not work

Reply:
@oudeismetis

Reply:
Ah I see your point, yeah the problem with JS/TS is that if for some reason you have anything that is surrounded with `{{  }}` it throws some templating error when executing cookiecutter. 

So I preferred to leave out the pages that don't have any cookiecutter templating syntax.

It was fine before, since none of the files within `/pages`  had templating syntax there, but now I need those two to be included

Reply:
ahh yes that's true!

Reply:
It seems there was only one file that needed it so I just escaped the variable with quotes and it worked. I also went ahead and updated the package.json and settings.py for the deployment.

Reply:
sounds like you figured it out.
For future reference I believe the cookiecutter docs have details on how to handle complicated situations like this.

Reply:
Yea but they do not offer a way to set negative look ups which sucks